### ISSUE TITLE
Issue on intro-teaser.ipynb

### ISSUE THREAD

**Issue by Prashiv2510** [tum-pbs/pbdl-book#23]
The code for dp part is as follows:

#Training
results_dp = nn_dp.fit(X, X, epochs=5, batch_size=5, verbose=1)

The second parameter of the fit method should have been Y in place of X i.e.  nn_dp.fit(X, Y, epochs=5, batch_size=5, verbose=1)

**Comment by caecfd** [comment]
However, once you designate it as Y, you won't get the desired results.

**Comment by thunil** [comment]
This is correct, the square (as "differentiable physics" operation) is taken in the loss function loss_dp().

The simpler version with Y is shown as "supervised" above in the notebook.

**Comment by caecfd** [comment]
您好，已收到您的邮件，我会在最短的时间内给您回复，谢谢！
